kube-merit-updater

```diff
@@ -261,14 +263,17 @@ cycle_nodes() {
   nodes=$(retry kubectl --context="${kube_context}" get nodes -l role="${role}",retiring="${retiring}" -o json |\
     jq -r '.items[].metadata.name')
   for node in ${nodes}; do
-    while [[ $(jobs -p | wc -l) -ge $max_nodes ]]; do
+    while [[ $(jobs | grep -v promtail | wc -l) -ge $max_nodes ]]; do
```

Comment:
I think it would be better to grep positive match if possible

Reply:
would the command to grep for be the function name `retry` in this case or `kubectl`

Reply:
Actually we are backgrounding all sorts of stuff in that `cycle_node` func, I take back my comment, negative match is better here.